google-cloud-bigquery/pom.xml

```diff
@@ -117,37 +116,6 @@
     <dependency>
       <groupId>org.apache.arrow</groupId>
       <artifactId>arrow-memory-netty</artifactId>
-      <!--
-      Due to JDK8 requirement, we are forced to use arrow version below v18
-      which has io.netty:netty-common a dep. and its vulnerabilities
-      -->
-      <exclusions>
-        <exclusion>
-          <groupId>io.netty</groupId>
-          <artifactId>netty-common</artifactId>
-        </exclusion>
-        <exclusion>
-          <groupId>io.netty</groupId>
-          <artifactId>netty-buffer</artifactId>
-        </exclusion>
-      </exclusions>
-      <scope>runtime</scope>
-    </dependency>
-    <!--
-    Define io.netty versions to override org.apache.arrow:arrow-memory-netty
-    transitive dependency versions which contains vulnerabilities.
-    -->
-    <dependency>
-      <groupId>io.netty</groupId>
-      <artifactId>netty-common</artifactId>
-      <version>${netty.version}</version>
-      <scope>runtime</scope>
-    </dependency>
-    <dependency>
-      <groupId>io.netty</groupId>
-      <artifactId>netty-buffer</artifactId>
-      <version>${netty.version}</version>
```

Comment:
We don't need this override anymore because the new arrow 17 doesn't bring a vulnerable netty.

Reply:
iiuc, arrow 17 was upgraded in last release and causing the failure observed in test? For future reference, can you add a ref to where this dependency is upgraded in the description?

Reply:
(I missed you were aware of the arrow upgrade)
I added more context (PRs) in the description.